lib/WIFI/WIFI.h

```diff
@@ -92,7 +92,7 @@ class WIFI
     bool connectStation();
 
     /** Timeout for WiFi connection. */
-    const unsigned long WIFI_TIMEOUT_MS = 30000;
+    const unsigned long WIFI_TIMEOUT_MS;
```

Comment:
Use "static" keyword for constant values and initialize them at definition, because they are valid for all instances.